### DISCUSSION TITLE
Issues/mistakes in 'original' parameter files ie CMT2,3,4,5,6,7,12,20,21,31,44

### DISCUSSION THREAD

**Discussion by jsclein-uaf** [ua-snap/dvm-dos-tem#709] [General]
sometimes a CMT might have a mistake, we might want to double check things like the LAI (over the 12 months) making sure that the trend is what it is supposed to be, 
for example, 
should lichen, moss, sphagnum and maybe other moss, value be 1.5 for all 12 months
double check the sedges, (this is PFT that I found mistakes in a couple PFTs
and there might be a couple more numbers that we want to spot check in ALL CMTs

why is this important
1. obviously for all the past and future runs
2.  also because we are using some of these pfts to as 'seed' values when creating new pfts so we want to be sure all is well with these valies

**Comment by jsclein-uaf** [comment]
ISSUE/MISTAKE 1
in CMT06
bcg_vegeation file 
aleaf   paramter for lichen and sphagnum is INCORRECT
is should be 0.7964, not 0.07964

ALSO, in the dimvegetation file
parameter 'nonvascular type' is 0, **BUT SHOULD be '1' for sphagnum**

**Comment by jsclein-uaf** [comment]
ISSUE/MISTAKE 2
dimvegetation file      see below, especially CMT20, but some others are odd too
these are ONLY mosses that I have looked

CMT1 and CMT2    moss
initial LAI and 12 months of static LAI are 9.7135

CMT3   moss
initial LAI and 12 months of static LAI are 0.3008

CMT5      Sphagnum
initial LAI and 12 months of static LAI are 1.8 initial, then 2.0 for remainder

CMT5      Sphagnum
initial LAI and 12 months of static LAI are 2.1 initial, then 1.5 for remainder
FEATHER is 1.9 initial, and 1.5 for static all 12 months

CMT7     Moss
initial LAI and 12 months of static LAI are 1.5

CMT12     Feather
initial LAI and 12 months of static LAI are 2.0

CMT20    Feather
initial LAI and 12 months of static LAI are 1.5  **BUT THE STATIC are INCORRECT (they vary too much)**
OtherMoss  **BUT THE STATIC are INCORRECT (they vary too much)**

CMT21     Feather, OtherMiss, Sphagnum
initial LAI and 12 months of static LAI are 2.0

CMT31     Sphagnum
initial LAI and 12 months of static LAI are 5.024
Moss  values are 0.483

CMT44     Feather
initial LAI and 12 months of static LAI are 2.0

**Comment by jsclein-uaf** [comment]
ISSUE/MISTAKE 3
file = dimvegetation
ONLY looking at lichens  initial and static (12 month) LAIs

CMT4  **ALL ARE INCORRECT**
CMT5  initial 1.8, static 2.8
CMT6  initial 2.0, static 1.5
CMT7  values are 1.5
CMT12  values are 2.0
CMT20   **THESE ARE ALL INCORRECT**
CMT21 initial  0.125 static 2.0
CMT44  values 2.0

**Comment by jsclein-uaf** [comment]
ISSUE/MISTAKE 4
file = dimvegetation
ONLY looking at SEDGES initial and static (12 month) LAIs

CMT4 **ALL ARE INCORRECT**
CMT20 **ALL ARE INCORRECT**   carex
CMT5 , look ok, but should double check
CMT6 , look ok, but should double check
CMT31, seem low?
CMT44, look ok, but should double check
CMT21 , look ok, but should double check   Carex

**Comment by hgenet** [comment]
Moss LAI for CMT3 should be low as moss PFT are minor in deciduous forest.
Let’s review these tomorrow if we have time and you can join the meeting.
Helene

**Comment by tobeycarman** [comment]
Hey folks we changed this from an issue to discussion. Ideas is that as we address each CMT we can create an issue and associated topic branch and PR for that issue.

**Comment by hgenet** [comment]
Here is another one: check the root to shoot ratio for sedges in CMT05 [ overestimation]

**Comment by jsclein-uaf** [comment]
HERE is another one,
Might not be too important, but I think these values should be either 1 or 0
see below for issue with CMT5 in dimveg file
// CMT05 // Tussock Tundra // (updated 1/15/2016 JDC)
//Betula    Decid       EGreen       Sedges       Forbs        Lichens      Feather      Sphag        PFT8         PFT9         // pftnames:
0.01        0.01        0.01         0.01         0.01         0.01         0.01         0.01         0.           0.           // vegfrac: vegetation coverage on land - 0 indicates that PFT doesn't exist

**Comment by jsclein-uaf** [comment]
Parameter in dim vegetation file
 **frprod[0]** is at 1 (or 0), should be 100
CMT1,2,3 moss (at 0)
CMT05 lichen feather sphagnum
CMT06 lichen feather sphagnum
CMT07 lichen moss
CMT12 feather lichen
CMT21 feather lichen othermoss sphagnum
CMT31 sphagnum moss   ARE AT 0
CMT44 lichen feather
CMT51 sphagnum lichen
CMT52 lichen              (joy will fix)
CMT60 lichen moss     at 0 (joy will fix)
CMT61 lichen             (joy will fix, NEED TO TELL BEN too)

**THESE ARE OK///**
CMT04  these are OK
CMT20 feather lichen othermoss, I think all are at 100
CMT50 moss lichen   they are at 100

**Comment by tobeycarman** [comment]
Should write a check that all PFTs that are not defined (i.e. named none, misc, misc. ,pft0, pft1, etc) have all zero values.

If `vegfrac` (cmt_dimvegetaton.txt) and/or `initvegc[l,w,r]` (cmt_bgcvegetation.txt) are set to zero, then in theory all the other parameter values could be non-zero and would be ignored, but this has not been exhaustively tested. Might be easier to write a few utility functions in param.py that can zero-out values...

**Comment by jsclein-uaf** [comment]
ANOTHER ISSUE for CMT3
tin the calibration file  - here is NO Nfall(0) for Moss, which there should be
and in dimvegetation file
ifperenniel should be 1 not 0
and
nonvascular type, is this sphagnum or feather moss????
